detect_secrets/plugins/high_entropy_strings.py

```diff
@@ -35,9 +35,7 @@ def analyze_string(self, string: str) -> Generator[str, None, None]:
                 # This occurs on the default regex, but not on the eager regex.
                 result = result[1]
 
-            entropy_value = self.calculate_shannon_entropy(result)
-            if entropy_value > self.entropy_limit:
-                yield result
+            yield result
```

Comment:
Can you leave a comment here too explaining that the entropy check moved to `analyze_line`? Because from the base class people are going to assume this is supposed to only yield secrets